content/guides/remote-development.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Remote Development
+menu:
+  main:
+    parent: 'guides'
+    weight: 90
+
+weight: 90
+---
+We provide a command line tool to help with using ContinuousPipe as a remote development environment.
+
+This helps to create, build and destroy remote environments and keep files in sync with the local filesystem.
+
+## Prerequisites
+
+You will need the following:
+
+ * A ContinuousPipe hosted project with the GitHub integration set up
+ * The project checked out locally
+ * The IP address, username and password to use for Kubenetes cluster
+ * rsync installed locally
+ * A [keen.io](https://keen.io) write token, project id and event collection name if you want to log usage stats
+
+Note: if the GitHub repository is not the origin of your checked out project then you will
+need to add a remote for that repository.
+
+## Installation
+
+OSX (64bit):
+
+```
+sudo curl https://continuouspipe.github.io/remote-environment-client/0.0.1/darwin-amd64.gz > cp-remote.gz
+gzip -d cp-remote.gz;
+mv cp-remote /usr/local/bin/cp-remote
+chmod +x /usr/local/bin/cp-remote
+```
+
+Dependencies: You need to have 'git', and 'rsync' installed and available in the shell where cp-remote runs
+
+Linux (64-bits):
+
+if your architecture is 32bit use linux-386.gz rather than linux-amd64.gz
+
+```
+sudo curl https://continuouspipe.github.io/remote-environment-client/0.0.1/linux-amd64.gz > cp-remote.gz
+gzip -d cp-remote.gz;
+mv cp-remote /usr/local/bin/cp-remote
+chmod +x /usr/local/bin/cp-remote
+```
+
+Dependencies: You need to have 'git', and 'rsync' installed and available in the shell where cp-remote runs
+
+Windows (64-bits):
+
+if your architecture is 32bit use windows-386.gz rather than windows-amd64.gz
+
+* Download https://continuouspipe.github.io/remote-environment-client/0.0.1/windows-amd64.gz
+* Extract cp-remote.gz
+* Move cp-remote.exe into your project folder
+
+Dependencies: You need to have 'git', and 'cwRsync' installed and available in your environment PATHS variable
+
+### Migrate to CP-Remote Go
```

Comment:
Is this relevant anymore? Would this be served better as a Note section as a reminder for existing users rather than confusing new users?